env.d/terraform.dist

```diff
@@ -18,6 +18,9 @@ OS_IDENTITY_API_VERSION=3
 # Administration configuration
 TF_VAR_key_pair=
 
+# Gateway configuration
+TF_VAR_gateway_webrtc_domain=
```

Comment:
Could there be a default value here (for instance `meet.jit.si` or _rendez-vous_) to simplify the workflow by giving the exact format